src/services/services.ts

```diff
@@ -335,36 +335,43 @@ module ts {
             var paramTag = "@param";
             var jsDocCommentParts: SymbolDisplayPart[] = [];
 
-            ts.forEach(declarations, declaration => {
-                var sourceFileOfDeclaration = getSourceFileOfNode(declaration);
-                // If it is parameter - try and get the jsDoc comment with @param tag from function declaration's jsDoc comments
-                if (canUseParsedParamTagComments && declaration.kind === SyntaxKind.Parameter) {
-                    ts.forEach(getJsDocCommentTextRange(declaration.parent, sourceFileOfDeclaration), jsDocCommentTextRange => {
-                        var cleanedParamJsDocComment = getCleanedParamJsDocComment(jsDocCommentTextRange.pos, jsDocCommentTextRange.end, sourceFileOfDeclaration);
-                        if (cleanedParamJsDocComment) {
-                            jsDocCommentParts.push.apply(jsDocCommentParts, cleanedParamJsDocComment);
-                        }
-                    });
-                }
+            ts.forEach(declarations,(declaration, indexOfDeclaration) => {
+                if (indexOf(declarations, declaration) === indexOfDeclaration) {
+                    // Already collected the doc comments from this declaration.
+                    // Eg. var a: Array<string> | Array<number>; a.length
+                    // The property length will have two declarations of property length coming 
+                    // from Array<T> - Array<string> and Array<number>

```

Comment:
Move this comment above the `if`, explain that the two declarations will *appear* to be the same declaration here even though they are from different instantiations of the same type.